### ISSUE TITLE
error while loading shared libraries

### ISSUE THREAD

**Issue by Paarash** [OPM/opm-common#756]
Hello,

When I was using OPM today , got an upgrade alert from software updater. After the upgrade and install was done I cannot use flow anymore and I encounter an error shown below:
```
flow: error while loading shared libraries: libopmcommon.so.2018: cannot open shared object file: No such file or directory
```
I got tired of searching for the solution in the Internet. Someone told me to locate libopmcommon and the result was:
```
/usr/lib/x86_64-linux-gnu/libopmcommon.so.2019
/usr/lib/x86_64-linux-gnu/libopmcommon.so.2019.04
```
Then write the below code:
```
sudo ln -s /usr/lib/x86_64-linux-gnu/libopmcommon.so.2019 /usr/lib/x86_64-linux-gnu/libopmcommon.so.2018
```
When I've done this I encounter symbol lookup error. Any idea what the problem is and what should I do???

Thanks

**Comment by atgeirr** [comment]
This problem is due to the new release process being halted in the middle of updating since two significant regressions were found. The regressions have been fixed, and release 2019.04 will be available soon. In the meantime, please be patient. Sorry for your troubles!

**Comment by Paarash** [comment]
Thanks for your response.
Best regards
